### PR TITLE
Basic reverse proxy server setup

### PR DESCRIPTION
**Pull request by nicofh**
Basic gem, route and controller configuration setup.

### CODE REVIEW

**Comment by matiasmansilla1989** [comment]
@nicofh We should use `develop` as the main branch during development https://github.com/rootstrap/tech-guides/tree/master/git#develop-branch-develop-required

File: app/controllers/proxies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def index
+    # Assuming the WordPress server is being hosted on port 8080
+    reverse_proxy "http://kamihome.com" do |config|
+      # We got a 404!
+      config.on_missing do |code, response|
+        redirect_to "/" and return
+      end
+
+      # There's also other callbacks:
```
Comment (matiasmansilla1989) [inline app/controllers/proxies_controller.rb]: Remove commented code

File: test/controllers/proxies_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (matiasmansilla1989) [inline test/controllers/proxies_controller_test.rb]: Remove this file if the test is not doing anything.

**Comment by glm4** [comment]
Hi @nicofh,
just a small comment: The name of the repository should also follow our naming conventions, in this case  `yxorp-api`

**Pull request closed without merging**